### PR TITLE
fix #5 create Generic Grid for curvilinear grid

### LINKED ISSUE #5: cdsm2 createGenericGrid need to be generic
_From @doutriaux1 on August 9, 2016 16:18_

``` python
f2=  cdms2.open(fnmcurv)
In [4]: s=f2("sample")
In [5]: g=s.getGrid()
In [6]: lat = g.getLatitude()
In [7]: lon=g.getLongitude()
In [9]: g2 = cdms2.createGenericGrid(lat,lon)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-9-da081e424d79> in <module>()
----> 1 g2 = cdms2.createGenericGrid(lat,lon)
```

_Copied from original issue: UV-CDAT/uvcdat#2081_

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

File: Lib/axis.py [hunk 1]
```diff
@@ -1465,12 +1465,12 @@ def genGenericBounds(self, width=1.0):
             bnds = numpy.array([self[0]-delta,self[0]+delta])
 
         # Transform to (n,2) array
-        retbnds = numpy.zeros((len(ar),2),numpy.float64)
-        retbnds[:,0] = bnds[:-1]
-        retbnds[:,1] = bnds[1:]
+        retbnds = numpy.zeros((ar.shape+(2,)),numpy.float64)
```
Comment (doutriaux1) [inline Lib/axis.py:1468]: That's neat! I never realized you could do that.

File: tests/test_gen_grid.py [hunk 2]
```diff
@@ -6,9 +6,24 @@
 
 import cdms2, numpy, os, sys
 import basetest
+import pdb
```
Comment (doutriaux1) [inline tests/test_gen_grid.py]: please remove

**Review by doutriaux1 — CHANGES_REQUESTED** [review]

File: tests/test_gen_grid.py (hunk 2 above)
Comment (dnadeau4) [reply tests/test_gen_grid.py]: Done

File: Lib/axis.py (hunk 1 above)
Comment (dnadeau4) [reply Lib/axis.py:1468]: Still learning Eh?

**Pull request merged**